sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ if(WIN32)
     target_compile_options(azure-storage-common PUBLIC /wd28204 /wd28020)
 else()
     find_package(LibXml2 REQUIRED)
-    target_include_directories(azure-storage-common PRIVATE ${LIBXML2_INCLUDE_DIRS})
+    target_include_directories(azure-storage-common SYSTEM PRIVATE ${LIBXML2_INCLUDE_DIRS})
```

Comment:
https://github.com/Azure/azure-sdk-for-cpp/issues/3167

In this case, we didn't want to apply the "doxygen doc correctness" warnings as errors to libxml2, but we do want it for our own SDK (i.e. `-Wdocumentation-unknown-command`):
https://github.com/Azure/azure-sdk-for-cpp/blob/fda88bf7b8389575b9d973821efc604442e10493/cmake-modules/AzureGlobalCompileOptions.cmake#L27